config/locales/devise.en.yml

```diff
@@ -2,6 +2,12 @@
 
 en:
   devise:
+    two_factor_authentication:
+      success: "Two factor authentication setup successful."
+      attempt_failed: "Six digit code is not valid"
+      max_login_attempts_reached: "Access completely denied as you have reached your attempts limit"
```

Comment:
I feel like we should get this copy reviewed before we tell users their access has been **COMPLETELY DENIED** :smile:

Reply:
This isn't actually used at the moment I think.

Reply:
I've remove the unused lines